DER/DerivativesContracts/Options.rdf

```diff
@@ -607,6 +522,52 @@
 		<skos:definition xml:lang="en">seller of the underlying asset in an option transaction</skos:definition>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;VanillaOption">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;Option"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-gty;hasGuarantor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-der-der-opt;OptionGuarantor"/>
```

Comment:
Is this needed? Would it be the market maker or broker?

Reply:
@rivettp We can discuss that - it was in MB's model originally.